platform/wismote/Makefile.wismote

```diff
@@ -41,6 +41,17 @@ endif
 contiki-$(TARGET).a: ${addprefix $(OBJECTDIR)/,symbols.o}
 #	$(AR) rcf $@ $^
 
+SERIALDUMP = $(CONTIKI)/tools/sky/serialdump-linux
+MOTELIST = $(CONTIKI)/tools/zolertia/motelist-zolertia
+MOTES = $(shell $(MOTELIST) -c 2>&- | cut -f 2 -d ,)
```

Comment:
Assignment statements that have `$(shell )` or `$(wildcard )` functions in them should use the "simply expanded" assignment operator (`:=`) instead of the "recursively expanded" assignment operator (`=`):
```make
MOTES := $(shell $(MOTELIST) -c 2>&- | cut -f 2 -d ,)
```